OpenMLDB/src/src/main/java/com/_4paradigm/benchmark/Util.java

```diff
@@ -64,10 +64,13 @@ public static boolean executeSQL_block(String sql, SqlExecutor executor) {
                 System.out.println("[INFO LOG]:" + resOfShow.getString(6) + "///" + resOfShow.getString(3));
                 resOfShow.close();
                 TimeUnit.SECONDS.sleep(30);
-                retOfShow = state.execute("SHOW JOB " + jobID);
+                retOfShow = state.execute("SHOW JOB " + jobID + ";");
```

Comment:
`SHOW JOB` is a select from job_info table, if job_info table is broken(exists but empty), we may get empty resultSet. You should handle it. I think `hasNext()` is false in that situation.
And to be more robust, you shouldn't do endless while, set a max wait time, e.g. 2 hour or else.

Reply:
If you add a ut about `executeSQL_block`, I can produce the broken table to test it.

Reply:
> `SHOW JOB` is a select from job_info table, if job_info table is broken(exists but empty), we may get empty resultSet. You should handle it. I think `hasNext()` is false in that situation. And to be more robust, you shouldn't do endless while, set a max wait time, e.g. 2 hour or else.

Haven't fix this.